tor_ocr/main.py

```diff
@@ -76,8 +67,33 @@ def chunks(s, n):
         yield s[start:(start + n)]
 
 
-def run(config):
+def ocr_space_url(url, overlay=False, api_key=os.getenv("OCR_API_KEY", "helloworld"), language='eng'):
+    """ OCR.space API request with remote file.
+        Python3.5 - not tested on 2.7
+        This code is stolen from https://github.com/Zaargh/ocr.space_code_example/blob/master/ocrspace_example.py
```

Comment:
re: code length: this line is 113 characters long 😛

Reply:
I must have changed PyCharm’s settings 🙃